src/Providers/Stripe/Service.php

```diff
@@ -24,25 +29,40 @@ public function __construct(string $key)
      *
      * @param string $token
      * @param int $amount
+     * @param string|null $customerId
+     * @param string|null $description
+     * @param array $metadata
      * @param string $currency
      *
      * @return Payment
      *
-     * @throws Exception
+     * @throws PaymentException
      * @throws StripeException
      */
-    public function create(string $token, int $amount, string $currency = 'GBP'): Payment
-    {
+    public function create(
+        string $token,
+        int $amount,
+        ?string $customerId = null,
```

Comment:
nullable types with `?` 👌  nice - haven't seen before, but I'd prefer it to always be a string if it is passed, we should never really be passing `null` as an argument here IMO

Reply:
also when is this passed? on second payments?